http/src/main/resources/init-resources/startup.sh

```diff
@@ -243,16 +247,9 @@ MEM_LIMIT=${MEM_LIMIT}
 SHM_SIZE=${SHM_SIZE}
 END
 
-        COMPLETE_JUPYTER_DOCKER_COMPOSE="-f $JUPYTER_DOCKER_COMPOSE"
```

Comment:
I moved this up so it can also be applied to RStudio and Dataproc that also use GPUs